CyFile-Android/app/src/main/java/at/tugraz/tc/cyfile/CyFileApplication.java

```diff
@@ -24,12 +28,58 @@ public void onCreate() {
 
         mNoteComponent = DaggerNoteComponent
                 .builder()
-                .noteModule(new NoteModule(new SecureNoteService(new InMemoryNoteRepository(), new NoOpCryptoService())))
+                .noteModule(new NoteModule(new SecureNoteService(
+                        new InMemoryNoteRepository(CyFileApplication.getInitialNotes()), new NoOpCryptoService())))
                 .build();
-
     }
 
     public NoteComponent getNoteComponent() {
         return mNoteComponent;
     }
+
+    public static Set<Note> getInitialNotes() {
```

Comment:
I think making this method should be private and non-static because we will use this only for initial development and It should not be used at all across the whole app.